cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/KafkaCruiseControlConfig.java

```diff
@@ -244,6 +252,33 @@ private void sanityCheckConcurrency() {
                                 + "] cannot be greater than the maximum number of allowed movements in cluster ["
                                 + maxClusterPartitionMovementConcurrency + "].");
     }
+
+    int concurrencyAdjusterMinPartitionMovementsPerBroker = getInt(ExecutorConfig.CONCURRENCY_ADJUSTER_MIN_PARTITION_MOVEMENTS_PER_BROKER_CONFIG);
+    if (interBrokerPartitionMovementConcurrency < concurrencyAdjusterMinPartitionMovementsPerBroker) {
+      throw new ConfigException("Inter-broker partition movement concurrency [" + interBrokerPartitionMovementConcurrency
```

Comment:
Nit: I would use `String.format` to construct the exception messages to improve readability since these variable names are long. It applies to all `ConfigException` in this section.